src/main/kotlin/lotto/LottoNumber.kt

```diff
@@ -1,11 +1,15 @@
 package lotto
 
-class LottoNumber(val value: Int) {
+class LottoNumber(val value: Int) : Comparable<LottoNumber> {
     init {
         require(isInRange(value))
     }
 
     private fun isInRange(value: Int): Boolean {
         return value in 1..45
     }
+
+    override fun compareTo(other: LottoNumber): Int {
+        return value - other.value
```

Comment:
```suggestion
        return value.compareTo(other.value)
```

What could be the difference? I mean, they both look like subtraction right?
Although this won't affect lotto number comparison, it does have a difference.
It's an interesting topic. 😋